src/pytest_dbfixtures/factories/elasticsearch.py

```diff
@@ -109,6 +109,11 @@ def finalize_elasticsearch():
             shutil.rmtree(home_path)
 
         request.addfinalizer(finalize_elasticsearch)
+
+        def pytest_runtest_setup(item):
+            if elasticsearch_executor.running() == False:
```

Comment:
You are probably missing Pylama hook in the project.

`$ pylama --hook`

Then it won't let you commit code that doesn't lint.